tests/jsx-whitespace/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`test.js 1`] = `
+// Should collapse
```

Comment:
These shouldn't collapse. If it's out of scope for this PR, we could just change the comment.

http://babeljs.io/repl/#?babili=false&evaluate=false&lineWrap=false&presets=es2015%2Creact%2Cstage-2%2Cstage-3&targets=&browsers=&builtIns=false&debug=false&code=spaces%20%3D%20%3Cdiv%3E%5D%20%20%20%5B%3C%2Fdiv%3E&experimental=false&loose=false&spec=false&playground=true